templates/etc/mysql/galeranotify.py.j2

```diff
@@ -25,17 +25,18 @@ except ImportError:
     from email.MIMEText import MIMEText
 
 import socket
+import email.utils
 
 # Change this to some value if you don't want your server hostname to show in
 # the notification emails
 THIS_SERVER = socket.gethostname()
 
 # Server hostname or IP address
-mariadb_smtp_server = '{{ galera_notify_smtp_server }}'
+SMTP_SERVER = '{{ galera_notify_smtp_server }}'
 SMTP_PORT = {{ galera_notify_smtp_port }}
 
 # Set to True if you need SMTP over SSL
-SMTP_SSL = False
+SMTP_SSL = {{ galera_notify_smtp_ssl | default('False') }}
```

Comment:
I would suggest to define galera_notify_smtp_ssl in defaults/main.yml instead of setting its default value here. It is more convenient for role user to have it defined all together in one file.

Reply:
Ok, code updated as requested